### PR TITLE
Parameter description misplacement

### PR DESCRIPTION
**Pull request by cbagpipe**
## Description

- In `NetworkStream` constructor parameters descriptions was misplaced. Fixed it.

## Motivation and Context

- Avoid confusion.
- It's my first PR. So I highly motivated to train on this little fixes how to do PR's right.

## How Has This Been Tested?

- Checked by eyes.

## Types of changes

- [x] Documentation (changes or updates in the documentation, has no impact on code or features)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/main/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/nanoframework/System.Net?pullRequest=214) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/nanoframework/System.Net?pullRequest=214) before we can accept your contribution.<br/><br/>:x: cbagpipe [sign now](https://cla.dotnetfoundation.org/nanoframework/System.Net?pullRequest=214)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/nanoframework/System.Net?pullRequest=214) it.</sub>

**Review by networkfusion — COMMENTED** [review]
Looks good, but I would recommend also remove the excess namespace from the documentation as it (should) be obvious. (even though the original included it for some reason, or at least make it a doc ref)...

**Comment by cbagpipe** [comment]
@networkfusion, thanks. I will in future PR's. Now I have not enough experience to somehow edit this PR... commit... or what should I actually do to follow your advice?

**Comment by networkfusion** [comment]
> @networkfusion, thanks. I will in future PR's. Now I have not enough experience to somehow edit this PR... commit... or what should I actually do to follow your advice?

Just change and commit to your source branch and this PR will update automagically....

**Comment by cbagpipe** [comment]
@networkfusion, done!

File: nanoFramework.System.Net/Sockets/NetworkStream.cs [hunk 1]
```diff
@@ -86,13 +86,13 @@ public NetworkStream(Socket socket)
...
-        /// System.Net.Sockets.Socket with the specified System.Net.Sockets.Socket ownership.
+        /// Initializes a new instance of the <see cref="NetworkStream"/> class for the specified 
+        /// <see cref="Socket"/> with the specified <see cref="Socket"/> ownership.
         /// </summary>
-        /// <param name="socket">true to indicate that the System.Net.Sockets.NetworkStream will take ownership of the System.Net.Sockets.Socket; 
-        /// otherwise, false.</param>
-        /// <param name="ownsSocket">The System.Net.Sockets.Socket that the System.Net.Sockets.NetworkStream will
+        /// <param name="socket">The <see cref="Socket"/> that the <see cref="NetworkStream"/> will
         /// use to send and receive data.</param>
+        /// <param name="ownsSocket">true to indicate that the <see cref="NetworkStream"/> will take ownership of the <see cref="Socket"/>;
```
Comment (josesimoes) [inline nanoFramework.System.Net/Sockets/NetworkStream.cs:94]:
```suggestion
        /// <param name="ownsSocket"><see langword="true"/> to indicate that the <see cref="NetworkStream"/> will take ownership of the <see cref="Socket"/>; 
```

File: nanoFramework.System.Net/Sockets/NetworkStream.cs [hunk 2]
```diff
@@ -86,13 +86,13 @@ public NetworkStream(Socket socket)
...
+        /// Initializes a new instance of the <see cref="NetworkStream"/> class for the specified 
+        /// <see cref="Socket"/> with the specified <see cref="Socket"/> ownership.
         /// </summary>
-        /// <param name="socket">true to indicate that the System.Net.Sockets.NetworkStream will take ownership of the System.Net.Sockets.Socket; 
-        /// otherwise, false.</param>
-        /// <param name="ownsSocket">The System.Net.Sockets.Socket that the System.Net.Sockets.NetworkStream will
+        /// <param name="socket">The <see cref="Socket"/> that the <see cref="NetworkStream"/> will
         /// use to send and receive data.</param>
+        /// <param name="ownsSocket">true to indicate that the <see cref="NetworkStream"/> will take ownership of the <see cref="Socket"/>; 
+        /// otherwise, false.</param>
```
Comment (josesimoes) [inline nanoFramework.System.Net/Sockets/NetworkStream.cs:95]:
```suggestion
        /// otherwise, <see langword="false"/>.</param>
```

**Review by josesimoes — APPROVED** [review]
Thanks for fixing this. I just added a couple more fixes to it. 😉

**Pull request merged**